styles.css

```diff
@@ -1285,13 +1285,13 @@ h1 {
 }
 
 .btn:hover {
-  text-decoration-line: underline;
   background-color: #6070ff;
+  color: white;
 }
 
 .btn:active {
-  opacity: 1;
-  color: #5E6C84;
+  background-color: #2230d2;
+  color: white;
 }
```

Comment:
[OPTIONAL] Please consider removing the border from the button when it is being `hovered or clicked` to make it in accordance with the design. Moreover consider reviewing the width and height of the button also.

## Solution:-

```
.btn:hover, .btn:active {
  border: none;
}
```

```
.btn {
    padding: 12px 16px;
    width: 142px;
    height: 48px;
    border-radius: 8px;
}
```